src/io/matpower.jl

```diff
@@ -496,21 +492,54 @@ function parse_matpower_data(data_string::String)
 
         elseif parsed_matrix["name"] == "dcline"
             dclines = []
+            warn("DC Line costs are not considered")
```

Comment:
Shouldn't this line occur in the block, `elseif parsed_matrix["name"] == "dclinecost"`, that way, users will only see it if the data file includes a cost block?